lib/src/subnav/subnav.component.ts

```diff
@@ -1,13 +1,12 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, HostBinding } from '@angular/core';
 
 @Component({
     selector: 'hc-subnav',
-    template: `<div class="subnav" [ngClass]="{'fixed-top': fixedTop}"><ng-content></ng-content></div>`,
+    template: `<ng-content></ng-content>`,
```

Comment:
I wanted to add padding to this and couldn't because the .subnav div was inside its parent and angular scopes the styling. Thus I moved everything to the root element